libs/ktem/ktem/reasoning/simple.py

```diff
@@ -711,7 +831,22 @@ def get_pipeline(cls, settings, states, retrievers):
             retrievers: the retrievers to use
         """
         prefix = f"reasoning.options.{cls.get_info()['id']}"
-        pipeline = cls(retrievers=retrievers)
+        example_path = (
+            Path(getattr(flowsettings, "KH_APP_DATA_DIR", ""))
+            / "rephrase_question_train.json"
+        )
+        examples = json.load(open(example_path, "r"))
+        examples = [
+            {"question": item["input"], "rewritten_question": item["output"]}
+            for item in examples
+        ]
+        pipeline = cls(
+            retrievers=retrievers,
+            rewrite_pipelines=[
+                FewshotRewriteQuestionPipeline.get_pipeline(examples=examples),
+                DecomposeQuestionPipeline.get_pipeline(),
+            ],
```

Comment:
Since now that we have a list of `rewrite_pipelines`, it should be made into a param like `retrievers`. Maybe also rename it to `rewriters` (or rename `retrievers` to `retrieving_pipelines`) for the sake of symmetry.

Also, it would be nice if you could make the list of `rewrite_pipelines` configurable in `flow_settings.py`.